docs/book/src/tasks/updating-machine-templates.md

```diff
@@ -27,9 +27,11 @@ The correct process for modifying an infrastructure machine template is as follo
 Once the new infrastructure machine template has been persisted, users may modify
 the object that was referencing the infrastructure machine template. For example,
 to modify the infrastructure machine template for the `KubeadmControlPlane` object,
-users would modify the `spec.infrastructureTemplate.name` field. For a `MachineDeployment`
-or `MachineSet`, users would need to modify the `spec.template.spec.infrastructureRef.name`
-field. In all cases, the `name` field should be updated to point to the newly-modified
+users would modify the `spec.infrastructureTemplate.name` field. For a `MachineDeployment`,
+users would need to modify the `spec.template.spec.infrastructureRef.name`
+field. A `MachineSet` references a machine template in the same way, but it should
+not be manually edited because it is managed by Kubernetes. In all cases, the `name`
```

Comment:
Sounds good overall, but I think the reason is that our MachineSet controller doesn't rollout changes to the ref of a MachinSet. Afaik it would only create new Machines with the new ref

Reply:
Right, a MachineSet can be modified "manually" as far as it's not owned by a MachineDeployment. May be clarify a bit:
>For a `MachineDeployment`, users would need to modify the `spec.template.spec.infrastructureRef.name`
field and the controller would orchestrate the upgrade by managing `MachineSets` pointing to the new and old references. In the case of a MachineSet with no MachineDeployment owner, if its template reference is changed, it will only affect upcoming Machines.

?

Reply:
I have updated it.